### PR TITLE
Add lookbook

### PR DESCRIPTION
**Pull request by emileechen**
pls review :syringe: :syringe: :syringe: :syringe: :syringe: :syringe:

### CODE REVIEW

**Comment by mngyuan** [comment]
it luk gud 4 now make moar tho

File: css/lookbook.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville);
+@import 'common';
+@import 'styleguide';
```
Comment (mngyuan) [inline css/lookbook.scss]: you just need to import common, common imports styleguide. also delete the @import font lines, common takes care of it for you

File: css/lookbook.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	flex: 1;
+	height: 30vh;
+	padding: 0;
+	background: $offwhite;
+	margin-right: 10px;
+	margin-bottom: 10px;
+	min-width: 20vw;
+}
+.cover {
+	object-fit: cover;
```
Comment (mngyuan) [inline css/lookbook.scss:53]: idk what this does but i trust u

File: html/lookbook.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		<link rel="stylesheet" type="text/css" href="../css/compiled/lookbook.css">
+	</head>
+
+	<body>
+		<div id="main">
+
+			<div class="desc">
+<!-- 				<h1>Title</h1> -->
+				<span class="date">March 2016</span><span class="words"> / InnoDxRetreat</span>
+				<!-- <div>Wow, what a cool description.</div> -->
```
Comment (mngyuan) [inline html/lookbook.html:12]: Wow!

File: css/lookbook.scss [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.nametag {
+	min-width: 0;
+	color: $offwhite;
+	background-color: $offblack;
+	font-family: 'Montserrat', sans-serif;;
```
Comment (mngyuan) [inline css/lookbook.scss]: semicolon and use $font-header

File: css/lookbook.scss [hunk 5]
```diff
@@ -0,0 +1,93 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville);
+@import 'common';
+@import 'styleguide';
+
+
+body, html {
+	background-color: $offwhite;
+	color: $offblack;
+	font-family: 'Open Sans', sans-serif;
```
Comment (mngyuan) [inline css/lookbook.scss]: does this line do anything? pls delete unless you want open sans

File: css/lookbook.scss [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+	font-size: 16px;
+	text-transform: uppercase;
+	font-family: 'Montserrat', sans-serif;
+}
+.wot{
+	position: relative;
+	top: -1em;
+	margin-bottom: 10px;
+	font-size: 10px;
+	font-family: 'Libre Baskerville', serif;
```
Comment (mngyuan) [inline css/lookbook.scss]: wait is this the italic'd version because then $font-accent

File: css/lookbook.scss [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+	position: relative;
+	top: 50%;
+	transform: translateY(-50%);
+	text-align: center;
+}
+
+.woo{
+	font-size: 16px;
+	text-transform: uppercase;
+	font-family: 'Montserrat', sans-serif;
```
Comment (mngyuan) [inline css/lookbook.scss]: $font-header

**Comment by mngyuan** [comment]
:hankey: :hankey: :hankey: 
fix small nit and we are gucci it looks :fire:

File: css/lookbook.scss [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+.nametag {
+	min-width: 0;
+	color: $offwhite;
+	background-color: $offblack;
+	font-family: $font-heading;
+	padding: 1em;
+	box-sizing: border-box;
+}
+
+.namee {
```
Comment (mngyuan) [inline css/lookbook.scss:65]: is this a typo

File: css/lookbook.scss [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+	text-align: center;
+}
+
+.woo{
+	font-size: 16px;
+	text-transform: uppercase;
+	font-family: $font-heading;
+	letter-spacing: .25em;
+}
+.wot{
```
Comment (mngyuan) [inline css/lookbook.scss:78]: i just noticed these class names wot LOL

**Comment by mngyuan** [comment]
Comrade :family:, your :hankey: is of the highest :100: quality. :sun_with_face: :+1: 

it has merge conflicts tho ugh :eyes: squash errythang and I'll deal with merging

[edit]
github added squashing nvm just try their squashing

File: css/lookbook.scss (hunk 9 above)
Comment (emileechen) [inline css/lookbook.scss:78]: i do what i want :nail_care:

Comment (mngyuan) [inline css/lookbook.scss:78]: whatevah whatevah, i do what i want

**Pull request merged**